app/src/main/java/com/nutomic/syncthingandroid/model/Folder.java

```diff
@@ -13,6 +13,7 @@
 
     public String id;
     public String label;
+    public String filesystemType = "basic";
```

Comment:
Why do you need this?
Does not having this (and other attributes you are adding below) this break something?

Reply:
I made a comparison with syncthing's v0.14.46 config.xml to the Folder model and found keys missing in there. I thought it is good practice to have them set to a default. This can't break anything because the model is normally retrieved by rest get "/rest/system/config" before any change is done to the attributes.  Reason why I put this is I stumbled over the problem with the fsWatcherEnabled in my previous PR where this wasn't saved correctly by syncthing core on the POST request because fsWatcherDelayS was missing in the model. Imagine a user updating syncthing app from a very old version before the fsWatcher (or filesystemType attribute). It could cause the config to break if the app assumes the REST API sent information on it but it didn't on the upgrade in place. Then a write-back the app's post could mess it up.

Reply:
Sure.